client/src/utils/data-formats/formats/bars.ts

```diff
@@ -59,25 +58,11 @@ export function bars(
   const valueField = args.valueField || 'count';
   const barsField = args.barsField || null;
   const seriesField = args.seriesField || null;
-  const valueMaxLength = args.valueMaxLength && parseInt(args.valueMaxLength, 10) || 13;
   const threshold = args.threshold || 0;
   const othersValue = args.othersValue || 'Others';
 
   let values: any[] = state.values || [];
 
-  // Concating values with '...'
-  if (values && values.length && valueMaxLength && (seriesField || barsField)) {
```

Comment:
Did you add those lines back? (Not sure since `if (values && values.length && valueMaxLength && (seriesField || barsField)) {` is still different :)